spec/fixtures/files/datahashes/core/collectionobject1.json

```diff
@@ -12,14 +12,11 @@
     "briefDescription": "Photo moto;chief inspector;",
     "distinguishingFeatures": "Verified account.",
     "comment": "Ndoto moto",
-    "computedCurrentLocationLocationLocal": "~Indeterminate Location~",
-    "computedCurrentLocationLocationOffsite": null,
-    "computedCurrentLocationOrganization": null,
```

Comment:
These fields shouldn't be included in the CSV template, so this is a fix to the cspace-config-untangler code I need to make! Issue recorded [here](https://github.com/collectionspace/cspace-config-untangler/issues/27)